example/service.twirp.go

```diff
@@ -137,7 +137,7 @@ func (s *haberdasherServer) writeError(ctx context.Context, resp http.ResponseWr
 // HaberdasherPathPrefix is used for all URL paths on a twirp Haberdasher server.
 // Requests are always: POST HaberdasherPathPrefix/method
 // It can be used in an HTTP mux to route twirp requests along with non-twirp requests on other routes.
-const HaberdasherPathPrefix = "/twirp/twitch.twirp.example.Haberdasher/"
+const HaberdasherPathPrefix = "/twitch.twirp.example.Haberdasher/"
```

Comment:
I wonder if it would be better to add a new constant name to make sure that the compiler tells the developer that there's something that needs to change after the upgrade. Since the "PathPrefix" now is just the service full name with `/`, maybe it would make sense to change the constant to this?
```go
const HaberdasherFullName = "twitch.twirp.example.Haberdasher"
```